src/amo/css/inc/mixins.scss

```diff
@@ -17,7 +17,14 @@
 @mixin font-family-inter() {
   font-family: 'Inter', sans-serif;
   @supports (font-variation-settings: normal) {
-    font-family: 'Inter Var', sans-serif;
+
+    /* default to font-display: swap version, but on larger screens, where the
+    flash of content would be more visible, use the blocking version. */
+    font-family: 'Inter Var Swap', sans-serif;
+
+    @include respond-to(extraExtraLarge) {
```

Comment:
One problem with doing this here is that every `mixin` that includes this one will have the media query. So the final CSS has this bit repeated quite a few times instead of having it consolidated in one rule. This was already a "problem" before - the `font-family` bit was often repeated - but now that the mixin is larger it ends up being a little bit too much and that's why the CSS has increased in file size.

If we care, we could change the various mixins to stop including the `font-family-inter()` and set the `font-family` only where needed - for most things inheriting from the `font-family` set on the `body` is enough - that would make the styles of the various components less independent though.

Reply:
I think that would be an enhancement. Is there any other font-family used on the website? Declare the font-family for body, form elements and the few other elements that would other fall back to the browser CSS.

Reply:
> If we care, we could change the various mixins to stop including the `font-family-inter()` and set the `font-family` only where needed - for most things inheriting from the `font-family` set on the `body` is enough - that would make the styles of the various components less independent though.

Yeah, that sounds like a good idea but for a different issue/patch (which I think should land before this one)